web-components/src/components/combobox/ComboBox.ts

```diff
@@ -793,7 +794,7 @@ export namespace ComboBox {
         <button
           type="button"
           class="md-combobox-button arrow-down"
-          aria-label=${this.label}
+          aria-label="Arrow Down"
```

Comment:
why hard coded ariaLabel? it should be passed from application. so that it can be localized